### DISCUSSION TITLE
Possible to use Heat Index target and work backward to a target temperature value?

### DISCUSSION THREAD

**Discussion by robinogi** [dolezsa/thermal_comfort#145] [Q&A]
If I decided, for example, that I wanted to keep my home at temperature where the Heat Index = 22 C, is it possible to determine what target temperature to set at the thermostat to achieve it?

**Comment by dolezsa** [comment]
I am not an expert on the subject, but I say in short: no (with only thermostat). or only if we can ensure a roughly constant humidity in the room. otherwise the problem is not impossible, but quite complex. since in our case the heat index is a function of 2 variables (temperature and humidity), both must be regulated while one affects the other...
